src/main/java/by/glinkiy/lesson2/task13/BiggestDigitInArray.java

```diff
@@ -0,0 +1,37 @@
+package by.glinkiy.lesson2.task13;
+
+import java.util.Random;
+/**
+ * Создать последовательность случайных чисел, найти и вывести наибольшее
+ * из них.
+ */
+
+public class BiggestDigitInArray {
+    public static void main(String[] args) {
+        int[] x = new int[10];
+        Random rand = new Random();
+        int a = rand.nextInt(20);
+        for (int i = 0; i < x.length; i++) {
+            x[i]= a;
+            a= rand.nextInt(20);
+
+        }
+//        for (int a1:x)
+//              {
+//                  System.out.println(a1);
```

Comment:
please remove commit, of course I expect to see initial numbers
        for (int num : arrayOfNumbers) {
            System.out.print(num + " ");
        }